ci-operator/step-registry/storage/conf/storageclass/set-default-storageclass/storage-conf-storageclass-set-default-storageclass-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: storage-conf-storageclass-set-default-storageclass
+  from: cli
+  # inject oc binary
+  cli: latest
+  commands: storage-conf-storageclass-set-default-storageclass-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: EXPECTED_DEFAULT_STORAGECLASS
+    documentation: |-
+      Name of expecting default storageclass.
```

Comment:
"Name of the storage class to set as the new default. The storage class must already exist."

Reply:
Addressed, thks!